### PR TITLE
Move selected tabs to new window intent

### PR DESCRIPTION
**Pull request by maitrella**
At the moment only apply for "highlighted" tabs via "Ctrl" key not for tabs previously selected via tabs query.

### CODE REVIEW

**Comment by maitrella** [comment]
@ianb I don't know if it's exactly what you want for #1102

File: extension/intents/tabs/tabs.toml [hunk 1]
```diff
@@ -226,3 +226,12 @@ match = """
   (Go | switch | change | activate |) (to |) (the |) first (pinned | app) tab [dir=first]
   (Go | switch | change | activate |) (to |) (the |) last (pinned | app) tab  [dir=last]
 """
+
+[tabs.moveSelectedToNewWindow]
+description = "Move selected tabs to (a |) new window"
+match = """
+  move selected tabs to new window
+"""
```
Comment (ianb) [inline extension/intents/tabs/tabs.toml]: Oops, you put the match pattern in `description` and the description in `match`

Some other phrases:

* `(launch | start |) new window (with |) selected tab{s}`
* `(move | launch | start) selected tabs (to | in |) (new |) window`

I guess that's all I can think of for now...

**Comment by ianb** [comment]
Yes, this looks right. Just a phrase matching tweak...

**Comment by maitrella** [comment]
Sorry, fixed.

**Pull request merged**